### ISSUE TITLE
LoadError: cannot load such file -- google/rpc/status_pb

### ISSUE THREAD

**Issue by aagarwal92** [GoogleCloudPlatform/google-cloud-ruby#1805]
Hi,

I'm attempting to run the google cloud vision libraries with ruby ruby 2.0.0p648

I have successfully added gem "google-cloud-vision" to my gemfile and run a proper bundle install. When trying to require "google/cloud/vision", I get the below stacktrace:

LoadError: cannot load such file -- google/rpc/status_pb
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:135:in `require'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:135:in `rescue in require'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:144:in `require'
	from /Library/Ruby/Gems/2.0.0/gems/grpc-1.6.7-universal-darwin/src/ruby/lib/grpc/google_rpc_status_utils.rb:16:in `<top (required)>'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Library/Ruby/Gems/2.0.0/gems/google-gax-0.8.12/lib/google/gax/grpc.rb:31:in `<top (required)>'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Library/Ruby/Gems/2.0.0/gems/google-gax-0.8.12/lib/google/gax/errors.rb:32:in `<top (required)>'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Library/Ruby/Gems/2.0.0/gems/google-gax-0.8.12/lib/google/gax/api_callable.rb:32:in `<top (required)>'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Library/Ruby/Gems/2.0.0/gems/google-gax-0.8.12/lib/google/gax.rb:30:in `<top (required)>'
... 4 levels...
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Library/Ruby/Gems/2.0.0/gems/google-cloud-vision-0.25.0/lib/google/cloud/vision/v1.rb:15:in `<top (required)>'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Library/Ruby/Gems/2.0.0/gems/google-cloud-vision-0.25.0/lib/google/cloud/vision/service.rb:18:in `<top (required)>'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Library/Ruby/Gems/2.0.0/gems/google-cloud-vision-0.25.0/lib/google/cloud/vision/project.rb:18:in `<top (required)>'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:73:in `require'
	from /Library/Ruby/Gems/2.0.0/gems/google-cloud-vision-0.25.0/lib/google/cloud/vision.rb:17:in `<top (required)>'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:135:in `require'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:135:in `rescue in require'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:144:in `require'
	from (irb):1
	from /usr/bin/irb:12:in `<main>'irb(main):002:0>

**Comment by blowmage** [comment]
@geigerj This looks like another file permission problem with the proto gems. I'm seeing files in the googleapis-common-protos 1.3.6 and googleapis-common-protos-types 1.0.0 gems that are missing global write permissions (`[-rw-r-----]`). So if the gem is installed by a different user in a different group it can't be used. This is the same problem discussed in GoogleCloudPlatform /google-cloud-ruby#1194 and googleapis/gax-ruby#54.

I am not aware of repos for these gems. Is this issue sufficient to report this, or should another issue be opened on another repo?

```
$ tree -p googleapis-common-protos-types-1.0.0/lib 
(...)/googleapis-common-protos-types-1.0.0/lib
└── [drwxr-xr-x]  google
    ├── [drwxr-xr-x]  api
    │   ├── [-rw-r-----]  annotations_pb.rb
    │   ├── [-rw-r-----]  auth_pb.rb
    │   ├── [-rw-r-----]  backend_pb.rb
    │   ├── [-rw-r-----]  billing_pb.rb
    │   ├── [-rw-r-----]  config_change_pb.rb
    │   ├── [-rw-r-----]  consumer_pb.rb
    │   ├── [-rw-r-----]  context_pb.rb
    │   ├── [-rw-r-----]  control_pb.rb
    │   ├── [-rw-r-----]  distribution_pb.rb
    │   ├── [-rw-r-----]  documentation_pb.rb
    │   ├── [-rw-r-----]  endpoint_pb.rb
    │   ├── [-rw-r-----]  http_pb.rb
    │   ├── [-rw-r-----]  httpbody_pb.rb
    │   ├── [-rw-r-----]  label_pb.rb
    │   ├── [-rw-r-----]  log_pb.rb
    │   ├── [-rw-r-----]  logging_pb.rb
    │   ├── [-rw-r-----]  metric_pb.rb
    │   ├── [-rw-r-----]  monitored_resource_pb.rb
    │   ├── [-rw-r-----]  monitoring_pb.rb
    │   ├── [-rw-r-----]  quota_pb.rb
    │   ├── [-rw-r-----]  service_pb.rb
    │   ├── [-rw-r-----]  source_info_pb.rb
    │   ├── [-rw-r-----]  system_parameter_pb.rb
    │   └── [-rw-r-----]  usage_pb.rb
    ├── [drwxr-xr-x]  logging
    │   └── [drwxr-xr-x]  type
    │       ├── [-rw-r-----]  http_request_pb.rb
    │       └── [-rw-r-----]  log_severity_pb.rb
    ├── [drwxr-xr-x]  longrunning
    │   ├── [-rw-r-----]  operations_pb.rb
    │   └── [-rw-r-----]  operations_services_pb.rb
    ├── [drwxr-xr-x]  rpc
    │   ├── [-rw-r-----]  code_pb.rb
    │   ├── [-rw-r-----]  error_details_pb.rb
    │   └── [-rw-r-----]  status_pb.rb
    └── [drwxr-xr-x]  type
        ├── [-rw-r-----]  color_pb.rb
        ├── [-rw-r-----]  date_pb.rb
        ├── [-rw-r-----]  dayofweek_pb.rb
        ├── [-rw-r-----]  latlng_pb.rb
        ├── [-rw-r-----]  money_pb.rb
        ├── [-rw-r-----]  postal_address_pb.rb
        └── [-rw-r-----]  timeofday_pb.rb

7 directories, 38 files
```


```
$ tree -p googleapis-common-protos-1.3.6/lib 
(...)/googleapis-common-protos-1.3.6/lib
└── [drwxr-xr-x]  google
    └── [drwxr-xr-x]  longrunning
        └── [-rw-r-----]  operations_services_pb.rb

2 directories, 1 file
```

**Comment by geigerj** [comment]
@blowmage Ack. We can track with this issue here. I will get a patch out for these gems.

**Comment by blowmage** [comment]
@aagarwal92 Thank you for opening the issue and letting us know of this problem. We are working on getting new gems released that will fix this for you. Until then, you can try uninstalling/reinstalling these two gems with the same user that ruby is running as. If that isn't an option, you can try to fix the file permissions directly using the approach described in [this comment](https://github.com/GoogleCloudPlatform/google-cloud-ruby/issues/1194#issuecomment-274108571).

**Comment by geigerj** [comment]
@aagarwal92 I have released `googleapis-common-protos-1.3.7` and `googleapis-common-protos-types-1.0.1`. Please upgrade to the latest versions, and re-open if the problem persists.
